app/controllers/bulk_upload_lettings_logs_controller.rb

```diff
@@ -51,7 +53,7 @@ def form
               when "needstype"
                 Forms::BulkUploadLettings::Needstype.new(form_params)
               when "upload-your-file"
-                Forms::BulkUploadLettings::UploadYourFile.new(form_params.merge(current_user:))
+                Forms::BulkUploadLettings::UploadYourFile.new(form_params.merge(current_user:, request:))
```

Comment:
Pass the request to this page so that I can set the back link based off the organisation id.